binsync/extras/__init__.py

```diff
@@ -9,6 +9,7 @@
 try:
     # AI Extras
     importlib.import_module("dailalib")
+    _l.info("Extras are available")
```

Comment:
Uneeded, as this will be printed each time people do `import binsync`. Please remove or turn it into `_l.debug`.